CodeGen/templates/Action.h

```diff
@@ -20,7 +20,6 @@ struct {{data.ModuleAPI}} F{{data.StructName}}SendGoalRequest
 	GENERATED_BODY()
 
 public:
-	UPROPERTY()
```

Comment:
`UPROPERTY()` removed? same at line 85, 141

Reply:
it seems bp do not support uint

Reply:
Oh so it is not compilable even without
BlueprintReadWrite?
Otherwise, all members should be tagged with UPROPERTY to be reset also per PIE

Reply:
yeah it failed eve with just `UPROPERTY()`
/home/yu/turtlebot3-UE/Plugins/rclUE/Source/rclUE/Public/Actions/ROS2FibonacciAction.h(24) : Error: Unrecognized type 'uint' - type must be a UCLASS, USTRUCT or UENUM